src/presentation/http/schema/NoteSettings.ts

```diff
@@ -24,5 +24,25 @@ export const NoteSettingsSchema = {
       maxLength: 10,
       minLength: 10,
     },
+    team: {
+      type: 'array',
+      items: {
+        type: 'object',
+        properties: {
+          id: {
+            type: 'number',
+          },
+          noteId: {
+            type: 'number',
+          },
```

Comment:
this is public id?

Reply:
No, it's internal Id

Reply:
i think we should not return internal id in router

Reply:
But we return it in GET /team route